crates/dapf/src/main.rs

```diff
@@ -293,6 +322,8 @@ enum Action {
     /// Interact with test routes behind `test-utils` feature flags.
     #[command(subcommand)]
     TestRoutes(TestAction),
+    /// Decode various payloads
```

Comment:
`various` could use some precision.
possibly also add an example command and how to retrieve/generate the payload from a leader/helper, maybe with dapf itself